code.js

```diff
@@ -0,0 +1,215 @@
+// Kane Kriz
+// UWYO COSC 3020 Algorithms - parallel mergesort primary code.js file
+// 10 May 2025
+//
+//
+
+
+
+
+
+//___________________________________________________________________________
+
+/**  
+ _____ DISCLAIMER: 
+ 
+ This implementation logically follows parallel merge sort  
+ ...but runs sequentially due to JS of course being single threaded and my intense difficulty getting paralleljs imported into the repo, file, or global environment in any capacity as was done in class. 
```

Comment:
You have to do something in parallel for this. You could use map reduce, which may be the easiest.